lib/build-json.js

```diff
@@ -124,17 +124,13 @@ const processLines = async () => {
       ...( await acc ),
       [ nameEnglish ]: await Promise.all( shabads.map( async shabad => ( { //! Do not destructure
       // Generate JSON with desired keys for Shabad
-        ...snakeCaseKeys( {
-          ...shabad,
-          writerId: undefined,
-          compositionId: undefined,
-          sectionId: undefined,
-          orderId: undefined,
-          subsectionId: undefined,
+        ...{
```

Comment:
You don't need to spread an object inside an object.

Also, was `snakeCaseKeys` changing the order? What was the culprit?

Would you be able to keep `snakeCaseKeys` and just not spread `...shabad`?

Reply:
`snakeCaseKeys` was working fine.... I don't know what was messing things. We can keep `snakeCaseKeys` but don't think we need it since we are hardcoding the keys

Reply:
Keep snakeCaseKeys, rebase it back on so that there are the same two commit you have right now, since snake_case is flagged up by ESLint so we tend to avoid it, and in the future, we are going to move to it fully.